hack/util.sh

```diff
@@ -641,7 +641,10 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
```

Comment:
Not a fan of this because it's very hard to think about what would happen if this contained a space (as it could on a dev box) ... forgetting the naked `$config` substitution, there is more confusion as the string gets passed into a function and then to `eval` ...